File: Latihan_B/php/Ram.php
<?php
class Ram {
    private $capacity;
    private $price;

    // constructor
    function __construct($capacity, $price) {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // method set dan get untuk capacity dan price
    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }
    function setPrice($price) {
        $this->price = $price;
    }

    function getCapacity() {
        return $this->capacity;
    }
    function getPrice() {
        return $this->price;
    }

    // method print
    function printRam() {
        echo "  Ram". "<br/>";
        echo "Capacity    : ". $this->getCapacity(). "<br/>";
        echo "Price       : ". "Rp. ". $this->getPrice(). "<br/>";
        echo "************************************". "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>

File: Latihan_B/java/Ram.java
public class Ram 
{
    private String capacity;
    private int price;

    // constructor
    public Ram(){}
    public Ram(String capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    // method set dan get untuk capacity dan price
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    public String getCapacity() {
        return capacity;
    }
    public int getPrice() {
        return price;
    }

    // method print
    public void printRam() {
        System.out.println("    Ram");
        System.out.println("Capacity    : " + capacity);
        System.out.println("Price       : " + "Rp. " + price);
        System.out.println("*************************************");
    }
}
